cmd/run.go

```diff
@@ -60,11 +60,11 @@ var runCommand = &cobra.Command{
 				volume = user.HomeDir + volume[1:]
 			}
 			dockerArgs = append(dockerArgs, "-v")
-			dockerArgs = append(dockerArgs, volume)
+			dockerArgs = append(dockerArgs, os.ExpandEnv(volume))
 		}
```

Comment:
I wonder if there are any security implications of expanding envvars here. We are trying to warn users about data that the command is going to use, and this is another vector for commands to steal data - #54